tests/conftest.py

```diff
@@ -290,6 +291,8 @@ def _assert_results_equal(
             result: The result data.
             rtol: The relative tolerance.
             atol: The absolute tolerance.
+            input_file_kwargs: Dictionary which contains the settings when extracting
+                the string from the input file.
```

Comment:
Maybe this makes more sense?
```suggestion
            input_file_kwargs: Dictionary which contains the settings when dumping the input file.
```